queries/angular/highlights.scm

```diff
@@ -21,47 +41,79 @@
   function: ((identifier) @function.builtin
     (#eq? @function.builtin "$any")))
 
+(pair
+  key: ((identifier) @variable.builtin
+  (#eq? @variable.builtin "$implicit")))
+
+((control_keyword) @repeat
+  (#any-of? @repeat "for" "empty"))
+
+((control_keyword) @conditional
+  (#any-of? @conditional "if" "else" "switch" "case" "default"))
+
+((control_keyword) @keyword.coroutine
+  (#any-of? @keyword.coroutine "defer" "placeholder" "loading"))
+
+((control_keyword) @exception
+  (#eq? @exception "error"))
+
+(special_keyword) @keyword
+
+((identifier) @boolean
+  (#any-of? @boolean "true" "false"))
+((identifier) @variable.builtin
+  (#any-of? @variable.builtin "this" "$event" "null"))
+
 [
-"let"
-"as"
-] @keyword
+ (ternary_operator)
+ (conditional_operator)
+] @conditional.ternary
 
 [
-"("
-")"
-"["
-"]"
-"{"
-"}"
+  "("
+  ")"
+  "["
+  "]"
+  "{"
+  "}"
+  "{{"
+  "}}"
+  "@"
+  "} @"
```

Comment:
This is weird.

Reply:
Yeah, but only "}" did not highlight those brackets used in `} @else {` for example.. Don't know why.

Reply:
Precedence problem in your grammar.

Reply:
Is it because I have `token(prec(2, '} @'))` in my grammar? If so, how would I resolve that? I could only get it to work using this.

Reply:
@amaanq ?

Reply:
Any ideas? Or can this PR be merged

Reply:
No ideas, sorry. I was hoping @amaanq (our resident parser expert) could help. Maybe ask on Matrix?

We can't merge right now anyway since master is frozen until the query changes are backported from `main` (ETA this week).

Reply:
The backports have been merged; please rebase on master, taking into account the changed valid capture names and the mandatory query formatting.